Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:14.4.0-alpine
+EXPOSE 8000
+WORKDIR /usr/defund12
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install --frozen-lockfile
+COPY . .
+ENTRYPOINT ["yarn"]
+CMD ["start-dev"]
```

Comment:
update CI to also use this

Reply:
shouldn't CI also use start given that the build results between it and dev are different?

Reply:
Right, have to think thru CI more (and learn about any differences in gatsby environments).. I think we just need to build assets and not even run gatsby since all the specs are just running against compiled files (not actually interacting over an http client w/a web driver).

to achieve this, we'd either override the CMD for the container -- or just pick the same base image and run `yarn install --frozen-lockfile` -- and then run specs in the container.